code/src/utest/pcgen/base/util/WeightedCollectionTest.java

```diff
@@ -146,7 +146,7 @@ public void testSimple() {
 		assertTrue(wc.add(I1));
 		assertFalse(wc.isEmpty());
 		assertTrue(wc.contains(I1));
-		assertTrue(wc.contains(new Integer(1))); // value semantic
```

Comment:
I think I noticed this and changed the way it gets tested in a different PR and forgot about it in  this one; closing